make-julia-latexsubs.jl

```diff
@@ -12,27 +12,40 @@ import REPL.REPLCompletions: latex_symbols, emoji_symbols
 Write Emacs lisp code that populates the hash table named `varname` to `dest`, using
 key-value pairs from `src`.
 """
-function write_latexsubs_hashtable(src, dest::IO, varname::AbstractString)
+function write_latexsubs_tables(io, src, hashvar, abbrevvar)
     for (k, v) in sort!(collect(src), by = last)
         ks = escape_string(k)
         vs = escape_string(v)
         if occursin(r"^\\U[0-9A-Fa-f]+$", vs)
             # codepoints outside the BMP can be problematic in older Emacsen
             cp = vs[3:end]
-            println(dest, "    (let ((c (decode-char 'ucs #x$cp)))\n",
-                    "      (if c (puthash \"$ks\" (char-to-string c) $(varname))))")
+            println(io, "    (let ((c (decode-char 'ucs #x$cp)))")
```

Comment:
While it may sound redundant to do `println` for each line, I love it how clean and easy-to-read this block ended up begin